libs/cdk/utils/directives/auto-complete/auto-complete.directive.ts

```diff
@@ -57,6 +57,9 @@ export class AutoCompleteDirective {
     /** @hidden */
     private lastKeyUpEvent: KeyboardEvent;
 
+    /** @hidden */
+    private isComposing = false;
```

Comment:
We usually prefix the private members with underscore. And I know the property above is private and not prefixed. We missed it during the review.